app/models/bus_stop.rb

```diff
@@ -208,9 +209,12 @@ def self.to_csv(limited_attributes: false)
     attrs = if limited_attributes
               LIMITED_ATTRS
             else
-              #binding.pry
               LIMITED_ATTRS.merge Hash[columns.reject do |v|
-                (v.name == 'name' || v.name == 'hastus_id' || v.name == 'updated_at' || v.name == 'route_list')
```

Comment:
I'd recommend using `column_names` and `except` to make this look a little nicer.